core/src/main/java/org/apache/servicecomb/core/exception/DefaultExceptionProcessor.java

```diff
@@ -0,0 +1,206 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@InjectProperties(prefix = "servicecomb.invocation.exception")
+public class DefaultExceptionProcessor implements ExceptionProcessor {
+  private static final Logger LOGGER = LoggerFactory.getLogger(DefaultExceptionProcessor.class);
+
+  public static final int ORDER = Integer.MAX_VALUE;
+
+  @SuppressWarnings("unchecked")
+  private final List<ExceptionConverter<Throwable>> converters = SPIServiceUtils
+      .getOrLoadSortedService(ExceptionConverter.class).stream()
+      .map(converter -> (ExceptionConverter<Throwable>) converter)
+      .collect(Collectors.toList());
+
+  @InjectProperty(
+      prefix = "servicecomb",
+      keys = {
+          "invocation.exception.print-stack-trace",
+          "exception.invocation.print-stack-trace"
+      },
+      defaultValue = "false")
+  protected boolean printStackTrace;
+
+  @InjectProperty(keys = "print-rate-limit", defaultValue = "false")
+  protected boolean printRateLimit;
+
+  private final Map<Class<?>, ExceptionConverter<Throwable>> converterCache = new ConcurrentHashMapEx<>();
+
+  @Override
+  public int getOrder() {
+    return ORDER;
+  }
+
+  @Override
+  public boolean isPrintStackTrace() {
+    return printStackTrace;
+  }
+
+  public DefaultExceptionProcessor setPrintStackTrace(boolean printStackTrace) {
+    this.printStackTrace = printStackTrace;
+    return this;
+  }
+
+  public boolean isPrintRateLimit() {
+    return printRateLimit;
+  }
+
+  public DefaultExceptionProcessor setPrintRateLimit(boolean printRateLimit) {
+    this.printRateLimit = printRateLimit;
+    return this;
+  }
+
+  @SuppressWarnings("unchecked")
+  @Autowired(required = false)
+  public DefaultExceptionProcessor setConverters(List<ExceptionConverter<? extends Throwable>> converters) {
+    converters.forEach(converter -> this.converters.add((ExceptionConverter<Throwable>) converter));
+    this.converters.sort(Comparator.comparingInt(ExceptionConverter::getOrder));
+    this.converterCache.clear();
+    return this;
+  }
+
+  @Override
+  public InvocationException convert(@Nonnull Invocation invocation, Throwable throwable) {
+    StatusType genericStatus = CONSUMER.equals(invocation.getInvocationType()) ? BAD_REQUEST : INTERNAL_SERVER_ERROR;
+    return convert(invocation, throwable, genericStatus);
+  }
+
+  @Override
+  public InvocationException convert(@Nullable Invocation invocation, Throwable throwable, StatusType genericStatus) {
+    Throwable unwrapped = ExceptionFactory.unwrap(throwable);
+    try {
+      return converterCache.computeIfAbsent(unwrapped.getClass(), clazz -> findConverter(unwrapped))
+          .convert(invocation, unwrapped, genericStatus);
+    } catch (Exception e) {
+      LOGGER.error("BUG: ExceptionConverter.convert MUST not throw exception, please fix it.\n"
+              + "original exception :{}"
+              + "converter exception:{}",

```

Comment:
add blank `converter exception` -> `converter exception `

Reply:
to align main content of original and converter